realsense2_camera/launch/rs_launch.py

```diff
@@ -109,7 +110,7 @@ def launch_setup(context, *args, **kwargs):
         return [
             launch_ros.actions.Node(
                 package='realsense2_camera',
-                namespace=LaunchConfiguration("camera_name"),
+                namespace=(LaunchConfiguration("camera_namespace"), "/", LaunchConfiguration("camera_name")),
```

Comment:
it is enough to do 
namespace=LaunchConfiguration("camera_namespace")

because it now has a default "camera" string inside it.
camera_namespace+camera_name ==> /camera/camera

In your code, it will result in node name = 
/camera/camera/camera